warehouse/oidc/forms.py

```diff
@@ -122,21 +114,22 @@ def _lookup_owner(self, owner):
 
         return response.json()
 
-    def validate_owner(self, field):
-        owner = field.data
-
-        # We pre-filter owners with a regex, to avoid loading GitHub's API
-        # with usernames/org names that will never be valid.
-        if not _VALID_GITHUB_OWNER.match(owner):
+    def validate_repo_slug(self, field):
+        if not _VALID_GITHUB_SLUG.match(field.data):
             raise wtforms.validators.ValidationError(
-                _("Invalid GitHub user or organization name.")
+                _("Invalid repository")
```

Comment:
Let's make this less generic:

```suggestion
                _("Invalid repository: not in user/repo format")
```